app/models/datawarehouse_node.rb

```diff
@@ -0,0 +1,9 @@
+class DatawarehouseNode < ApplicationRecord
```

Comment:
:+1: for moving this out of the global namespace. I am not sure if `Datawarehouse::Node` or `Datawarehouse::Elastic::Node` is better. I'm not sure what properties if any a `Datawarehouse::Elastic::Node` and a `Datawarehouse::Hawkular::Node` will share (if we ever decide to have something like that)